rewrite-maven/src/main/java/org/openrewrite/maven/search/ParentPomInsight.java

```diff
@@ -47,7 +43,14 @@ public class ParentPomInsight extends Recipe {
             example = "spring-boot-starter-*")
     String artifactIdPattern;
 
-    UUID searchId = randomId();
```

Comment:
Not sure what the `searchId` was for; Only saw that in FindProperties, where it's also unused and likely left over from https://github.com/openrewrite/rewrite/commit/392e1798c6d5b5695bb6524235f52ec40d429678 https://github.com/openrewrite/rewrite/blob/9965bbf5f26d3492100086d38cb9647071833108/rewrite-maven/src/main/java/org/openrewrite/maven/search/FindProperties.java#L44